src/relion/__init__.py

```diff
@@ -31,8 +34,10 @@ def __init__(self, path):
         :param path: A string or file system path object pointing to the root
                      directory of an existing Relion project.
         """
-        super().__init__("Import/job001")
         self.basepath = pathlib.Path(path)
+        super().__init__(
+            "Import/job001", locklist=[str(self.basepath / "default_pipeline.star")]
```

Comment:
String transforming here seems a bit clunky...
I know we have to do this eventually because of gemmi, but I'd prefer doing it as late as possible.
(Also: I've opened an issue with gemmi at https://github.com/project-gemmi/gemmi/issues/101)